controllers/volumereplication_controller.go

```diff
@@ -158,30 +164,52 @@ func (r *VolumeReplicationReconciler) Reconcile(ctx context.Context, req ctrl.Re
 		return ctrl.Result{}, nil
 	}
 
+	instance.Status.LastStartTime = metav1.Now()
```

Comment:
There can be a case that when you add `LastStartTime`, the resource already has the 'LastCompletionTime` from the previous reconcile. So we will have `LastStartTime` greater than `LasterCompletionTime` in the status.   So we should remove `LastCompletionTime` when we are updating the `LastStartTime`.

Reply:
Done